### ISSUE TITLE
Question about StandardOdk

### ISSUE THREAD

**Issue by Kharaxel** [emoose/xvdtool#26]
I see it being mentioned both here and when I do -wi on .xvd's from XONE's internal hdd. Question is... as I understand, we know it exists but it is private right? I just want to make sure. Because if it can be obtained, where do I get it from? If it's taboo then just gimme a hint. But if it's private, as I suspect, then nevermind. 
TL;DR: Can Key Extractor extract this from something? If this odk is private to M$, then nevermind.
Thank you, and I apologize if this is a dumb question.

**Comment by tuxuser** [comment]
Its a good question no worries.
Unfortunately, yes - the key is private and cannot be dumped so far.
As far as we know, StandardOdk is a 'variable' Id - the used key gets determined by Content Type Id. But we can assume, 90% of the times things are encrypted via GreenOdk

**Comment by Kharaxel** [comment]
Ah, I see. Alrighty then, thank you for your answer. : >
Microsoft really locked down this console hard. I don't care for "piracy", but there's some console exclusive games that I would like to poke into. Music/sfx digging, models etc. Still, maybe one day. Before I close this, let me just ask one more thing. I just tried mounting couple of the internal hdd's xvd's  (for giggles)and they apparently mounted a-okay... well, except that i'm getting access denied on them when trying to select it as gpt (or mbr for that matter). I assume this is normal behaviour for encrypted xvds, yes? This is the first time I "succesfully" mounted any so I'm just curious.

**Comment by tuxuser** [comment]
yeah, if no filesystem is detected, the files are likely broken or still encrypted.
If you like, post the "info" output.

**Comment by Kharaxel** [comment]
Thank you for the answers! I really appreciate it!

Here are results of writeinfo for two different xvds, because why not. The last line in them tells everything (can't display filesystem because they are encrypted).

710744C4-4CA5-4B8E-BAD1-B698DFB67C6B-PLS file info output:
https://pastebin.com/yeUqUfw0

ConnectedStorage-retail file info output:
https://pastebin.com/xDEqw37c

The file with the "PLS" at the end is of greatest interest to me. I assume its the "reserved space" from the game I'm particularly interested in. The reserved space is just cache as I understand it, of the most important files the game uses. But yeah. It might not be XVC like the main game files, yet its still done in with StandardODK.

Honestly, if I might ramble for a bit... this was quite a ride. Now that I understand everything (more or less) and that I know I won't get into the files, I don't feel bad. In fact, I feel quite accomplished. I learned a thing or two along the way. One of my greatest "oh snap" moments was learning about Base64 and finding out the license xml for the game (okay, it's Kingdom Hearts 3) have sections which is encoded in it. https://base64.guru/converter/decode This site of course helped me (and its encode/decode functions ha). Quite obvious to others, but to me it was quite interesting experience.
I apologize for the rambling, things like this really interest me even though I haven't tried learning programming and assemblies and other stuff. Makes me want to do so though. I think I just might.
Anyhoo, thank you once more. I'll leave this open for a bit in case you might bestow more wisdom upon me. : >

**Comment by ghost** [comment]
I presume there is still nothing on "StandardOdk"? I decided it would be fun to take a look at the Xbox One;s update files but can;t get anywhere so far. As mentioned before by Kharaxel, if it still can't be dumped, then nevermind.

**Comment by davFaithid** [comment]
Would it be possible to bruteforce the StandardOdk?

**Comment by arefdsg** [comment]
might be possible to obtain now, not completely sure, but as the readme says, if it is, there's no support for you if you use it obtained that way

**Comment by CactusDuper** [comment]
I know that there's no support provided for keys obtained from the leaks, but I'm curious if someone here has more info about those keys. From my conversations with some people I was told that the keys that got leaked were still encrypted but I'm not sure if that's true. If someone has more information about it, please let me know :)

**Comment by tuxuser** [comment]
[Deleted some comments]

**Comment by joeyoropesa-dev** [comment]
So... it's impossible to extract/mount and look at the filesystem of Xbox One (thanks to hard encryption of update files)....
